Traceback (most recent call last):
  File "/iitjhome/m22ma003/CV_Project/table_detection.py", line 274, in <module>
    train_dataset = TableDataset(annotation_file="table_data/train_data/labels/train/0_annotations.csv", image_dir="/content/table_data/train_data/images/train", transform=transforms.ToTensor())
  File "/iitjhome/m22ma003/CV_Project/table_detection.py", line 176, in __init__
    self.min_width, self.min_height = self.find_min_dimensions()
  File "/iitjhome/m22ma003/CV_Project/table_detection.py", line 213, in find_min_dimensions
    image = Image.open(image_path)
  File "/scratch/apps/anaconda3/lib/python3.9/site-packages/PIL/Image.py", line 3236, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/content/table_data/train_data/images/train/10_1_1_8_2198_11.jpg'
